contracts/rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "nightly-2022-03-14" # we override it here, due to compilation problems of cargo-contract using roots nightly version
```

Comment:
This is what I get compiling with the root pipeline (2022-05-23):

```
[1/5] Checking ink! linting rules
If you don't see a backtrace below, it could be because `cargo-dylint` wasn't built with a nightly compiler.
Error: 
Error: command failed: "rustup" "which" "rustc"
stdout: ""
stderr: "error: target triple in channel name 'nightly-2022-03-[14](https://github.com/Cardinal-Cryptography/aleph-node/runs/6864609484?check_suite_focus=true#step:13:15)-x86_64-unknown-linux-gnu'\n"
```

Full log: https://github.com/Cardinal-Cryptography/aleph-node/runs/6864609484?check_suite_focus=true

It pulled the right version but still tries to lint with the old one? Might be the caching in .cargo again, so I will try again with a forced re-install. See you in an hour.

Reply:
ok, worked after some battles with the pipeline